src/clients/storage/BlobStorageClient.js

```diff
@@ -0,0 +1,39 @@
+'use strict';
+
+let request = require('request');
+const apiUrlBase = process.env.FORTIS_CENTRAL_ASSETS_HOST || 'https://fortiscentral.blob.core.windows.net/';
```

Comment:
ok, the reason why I didn't do that was because of https://github.com/CatalystCode/fortisdeploy/blob/master/ops/setup-environment.sh#L22, but I'll look more into this